modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java

```diff
@@ -460,9 +471,15 @@ public void execute(Project p) {
 
 					setProperty.add(dockerImageId);
 
-					Property<String> property = dockerRemoveImage.getImageId();
+					Property<String> removeImageProperty =
```

Comment:
hey @Seiphon  can you sort this?  Get and Set the `pruneImageProperty` before the `removeImageProperty`
and lets call it
`Property<String> pruneImageIdProperty`
`Property<String> removeImageIdProperty`